src/styles/event-venue/frontend.pcss

```diff
@@ -76,3 +126,109 @@
 		min-height: 270px;
 	}
 }
+
+/* Theme specific overrides */
+
+/* Twenty-twenty-one theme */ 
+.theme-twentytwentyone .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: #334aff;
+}
+
+/* Twenty-twenty theme */
+.theme-twentytwenty {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__address,
+			.tribe-block__venue__address a,
+			.tribe-block__venue__phone,
+			.tribe-block__venue__website {
+				font-size: 1.65rem;
+			}
+
+			.tribe-block__venue__website a {
+				color: #334aff;
+			}
+		}
+	}
+}
+
+/* Divi */
+.theme-Divi {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__address,
+			.tribe-block__venue__phone,
+			.tribe-block__venue__website {
+				padding-bottom: 0; 
+			}
+
+			.tribe-block__venue__website a {
+				color: #334aff;
+			}
+		}
+	}
+}
+
+/* Genesis */
+.theme-genesis {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__address,
+			.tribe-block__venue__phone,
+			.tribe-block__venue__website {
+				font-size: 1.65rem;  
+			}
+
+			.tribe-block__venue__website a {
+				color: #334aff;
+			}
+		}
+	}
+}
+
+/* Avada */
+.theme-Avada {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			max-width: 80%;
+
+			@media screen and (min-width: 768px) {
+				max-width: 40%;
+			}
+
+			.tribe-block__venue__name {
+				.fusion-responsive-typography-calculated {
+					font-size: 1.2rem;
+					font-weight: normal;
+					letter-spacing: normal;    
+				}
+			}
+
+			.tribe-block__venue__address a,
+			.tribe-block__venue__website a {
+				color: #334aff;  
+			}
+		}
+
+		.tribe-block__venue__map {
+			@media screen and (min-width: 768px) {
+				max-width: 70%;
+			}
+		}
+	}
+}
```

Comment:
Left a note in slack about reaching out to Avada.

Reply:
@paulmskim lets wait and hear what the team has to say about this.